composer.json

```diff
@@ -77,5 +77,59 @@
             "ergebnis/composer-normalize": true
         },
         "sort-packages": true
+    },
+    "scripts": {
+        "cs:check": "php php-cs-fixer fix --dry-run",
```

Comment:
👎🏻 so , sometimes we need to use `dry-run` and sometimes `check`? confusing

Reply:
Personally sometimes I like to just display diff that would be applied to see if I can make some manual changes. I would keep it, but it needs `--diff` anyway.

Reply:
not sure how `diff` aspect factors in to `dry-run` vs `check` terms

Reply:
There are to `diff` contexts, so let's ensure we know what we're talking about:
- Git diff, used for `cs:diff` to analyse only changed files (it will be removed anyway)
- Diff result, printed to the CLI output

`cs:check` should make dry-run analysis on whole project and print the diff 🙂.